EXTCAP.md

```diff
@@ -0,0 +1,180 @@
+| Ubuntu     | 14.04 Trusty     |
+| Mac OS     | 10.11 El Capitan |
+| Windows 10 | 1803             |
+
+| Language  | Version          |
+|-----------|------------------|
+| Python    | 2.7.10           |
+
+| Software  | Version          |
+|-----------|------------------|
+| Wireshark | 3.0.3            |
+
+### Package Installation
+#### Automatic installation from source
+```
+git clone https://github.com/openthread/pyspinel
+cd pyspinel
+sudo python setup.py install 
+```
+#### Automatic install from PYPI (pending for new release package)
+```
+pip install pyspinel
+```
+#### Manual installation(for pyspinel 1.0.0a3 and before)
+**1. Install pyspinel package**
+``` 
+pip install pyspinel 
+```
+**2. Install extcap script on Wireshark**
+
+To find the correct installation path of the extcap utility on any system please open Wireshark:
+```
+"Help" -> "About Wireshark" -> "Folders" -> "Extcap path"
+```
+Copy the provided ```extcap_ot.py```, ```extcap_ot.bat``` and ```sniffer.py``` to the extcap directory.
+
+For OS X and Linux, verify that the ```extcap_ot.py``` file has the “x” permission. If not, add it using:
+
+```
+chmod +x extcap_ot.py
+```
+
+For Windows, verify that the ```extcap_ot.bat``` file is in the extcap folder
+
+## Usage
+
+### NAME
+    extcap_ot.py - external script for integrating OpenThread Sniffer with Wireshark
+
+### SYNOPSIS
+    extcap_ot.py [--arguments]
+
+### DESCRIPTION
+
+```
+    -h, --help            
+    	Show this help message and exit
+
+    --extcap-interfaces
+       	Provide a list of interfaces to capture from.
+
+    --extcap-interface <EXTCAP_INTERFACE>
+        Provide the interface to capture from.
+    
+    --extcap-dlts
+        Provide a list of dlts for the given interface
+
+    --extcap-config
+        Provide a list of configurations for the given interface.
+
+    --fifo <FIFO>
+        Use together with capture to provide the fifo to dump data to.
+    
+    --channel <CHANNEL>
+         IEEE 802.15.4 capture channel [11-26].
+
+    --baudrate <BAUDRATE>
+        Set the serial port baud rate.
+
+    --tap
+        Use to specify DLTs as IEEE 802.15.4 TAP (only for Wireshark3.0 and later).
+```
+
+## Quick Start
+### Configuring Wireshark for Thread
+* Wireshark configuration - [Protocols](https://openthread.io/guides/ncp/sniffer#wireshark_configuration_-_protocols)
+* Wireshark configuration - [FCS Format](https://openthread.io/guides/ncp/sniffer#wireshark_configuration_-_rssi)
+    *  For Wireshark 2.x: TI CC24xx metadata
+    * For Wireshark 3.x: ITU-T CRC-16
+   
+### Using the Sniffer
+#### Wireshark capture screen
+The Wireshark capture screen is displayed when Wireshark is first launched. 
+It includes the Wireshark interface for managing packets that are captured 
+and the hardware interfaces connected to the OT Sniffer.
+#### Start sniffing
+
+There are three ways to start sniffing:
+* If this is your first time using an interface, click on interface options 
```

Comment:
If referencing a control on a GUI, bold the name of the control and make sure it's formatted the same as in the GUI (icapitalized, all caps, etc).

Click on **Interface Options**
Click **Start**

Update as needed throughout the document.